restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/ClientsEndpoint.java

```diff
@@ -254,9 +255,15 @@ private void validate(final MultivaluedMap<String, String> data) throws RuntimeE
         } else if (!data.containsKey("Password")) {
             throw new NullPointerException("Password can not be null.");
         }
-        // https://github.com/RestComm/Restcomm-Connect/issues/1979
-        if (data.getFirst("Login").contains("@")) {
-            throw new IllegalArgumentException("Login contains invalid character: @ "+data.getFirst("Login"));
+        // https://github.com/RestComm/Restcomm-Connect/issues/1979 && https://telestax.atlassian.net/browse/RESTCOMM-1797
+        for (char ch: ClientLoginConstrains.NOT_ALLOWED_CHARS) {
```

Comment:
this should be reused from ClientLoginConstraints

Reply:
Agree